README.md

```diff
@@ -1,39 +1,39 @@
 # Peacefulli
 
-
 ## Prerequisites
 
-* Contact Aleyah Ewing or Daeton Den to create a peacefulli email address
-* Install [VS Code](https://code.visualstudio.com/download)
-* Install [git](https://git-scm.com/)
-  * Setup your config name and email `$ git config --global user.name "First Last" $ git config --global user.email "firstlast@peacefulli.com"`
-* Install [credential helper](https://www.atlassian.com/git/tutorials/install-git)
-* Install [node.js](https://nodejs.org/en/)
-* Install watchman `brew install watchman`
-* Install Expo `$ npm install --global expo-cli`
-* Install Yarn `$ brew install yarn`
-* Bookmark the [Jira Kanban](https://peacefulli.atlassian.net/wiki/spaces/PM/overview) board
+- Contact your manager to create a peacefulli email address
+- Install [VS Code](https://code.visualstudio.com/download)
+- Install [git](https://git-scm.com/)
+  - Setup your config name and email `$ git config --global user.name "First Last" $ git config --global user.email "firstlast@peacefulli.com"`
```

Comment:
nit: copy/paste of this is weird because it's two different commands, maybe we can split this like so:
`git config --global user.name "First Last"`
`git config --global user.email "firstlast@peacefulli.com"`

Reply:
fixing